examples/access_control/client_access_controll.c

```diff
@@ -0,0 +1,72 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+ * See http://creativecommons.org/publicdomain/zero/1.0/ for more information. */
+
+/**
+ * Using access_control_server
+ */
+
+#include <stdio.h>
+
+#ifdef UA_NO_AMALGAMATION
```

Comment:
only use the open62541.h in examples.

Refer to other examle files to see how it is used there.